src/compat/posix/proc/exit/exit.c

```diff
@@ -33,7 +33,11 @@ void _exit(int status) {
 	panic("Returning from _exit");
 }
 
+/* from atexit */
+void __atexit_funcs(void);
+
 /* stdlib */
 void exit(int status) {
+	__atexit_funcs();
```

Comment:
You should also add to Mybuild depends from atexit_api where __atexit_funcs is implemented

Reply:
done.